app/src/main/res/layout/fragment_web_container.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <LinearLayout
+    <FrameLayout
```

Comment:
안드로이드에서 `FrameLayout`은 `오직 하나의 자식 뷰 위젯만을 표시하도록 권고`하고 있는데 FrameLayout을 쓰신 이유가 혹시 있으신가요?

Reply:
@RidiSangHoon 
> FrameLayout is designed to block out an area on the screen to display a single item. Generally, FrameLayout should be used to hold a single child view, because it can be difficult to organize child views in a way that's scalable to different screen sizes without the children overlapping each other. You can, however, add multiple children to a FrameLayout and control their position within the FrameLayout by assigning gravity to each child, using the android:layout_gravity attribute.

https://developer.android.com/reference/android/widget/FrameLayout

말씀하신바를 처음들어서 😂 찾아보니,
겹쳐지는 방식으로 **레이아웃을 scalable하게 구성하기가 어렵기** 때문에 하나의 자식을 사용하라고 권고를 하고 있네요!

이 레이아웃의 경우엔,
1. WebView위에 ToolboxView가 겹쳐보여야함.
2. ToolboxView와 WebView 둘다 scalable함.

에 해당되어서 권고된 내용에 해당하지 않을것같습니다.